pyvrp/read.py

```diff
@@ -149,11 +149,13 @@ def read(
         durations = distances
         time_windows: np.ndarray = round_func(instance["time_window"])
     else:
-        # No time window data. So the time window component is not relevant,
-        # and we set all time-related fields to zero.
+        # No time window data. So the time window component is not relevant.
         durations = np.zeros_like(distances)
         service_times = np.zeros(dimension, dtype=int)
```

Comment:
Setting durations/service times to zero when there are no time windows no longer makes sense with vehicle max duration and shifts. We don't have any instances where it can go wrong, but it'd be convenient if we changed this now. Can we:
- always set `distances = durations` (VRPLIB-style)
- and leave `service_times` unmodified here?